src/spec/sort_spec.js

```diff
@@ -1,16 +1,13 @@
 import chai from 'chai'
 import sort from '../lib/sort.js'
-import R from 'ramda'
 const expect = chai.expect
 
 describe('Sort', () => {
   it('Sorts the list in ascending order based on difference function', () => {
```

Comment:
As long as this test is testing the right functionality and passes, cool. But I am unsure what this test is saying. Can you explain?